WINDOWS.md

```diff
@@ -0,0 +1,136 @@
+# Installing Chiadog on Windows
+
+## Pre-requisites
+
+- Linux, MacOS & Windows
+- [Python 3.7+](https://www.python.org/downloads/windows/)
+- [Git](https://git-scm.com/downloads)
+- Enabled `INFO` logs on your chia farmer
+
+This guide contains instructions on how to set up a **local** `chiadog` consumer on Windows.
+If you are looking for instructions on how to set up remote monitoring on a Windows machine, 
+please refer to [this article](/wiki/Monitoring-Multiple-Harvesters) 
+instead.
+
+Looking for installation instructions for Linux or MacOS? Head over to the 
+general [README](README.md).
+
+## How to enable INFO logs on chia farmer?
+
+First we'll set Chia's log level to `INFO`. This ensures that all logs necessary for `chiadog` to operate 
+are available under `C:\Users\[YOUR-USER]\.chia\mainnet\log\debug.log`.
+
+1. Open the file `C:\Users\[YOUR-USER]\.chia\mainnet\config\config.yaml` in your favorite text editor
+2. Find the line that reads `log_level: DEBUG` (under the `farmer` section) and change this to `log_level: INFO`
+3. Restart the GUI or run `chia start --restart farmer` from the command line
+
+## Installation
+
+_For updating from previous version, see section below._
+
+1. Open a `PowerShell` command line
+
+2. Clone the repository somewhere on your computer (ex. `C:\Users\[YOUR-USER]\`)
+
+```
+cd C:\Users\[YOUR-USER]\
+git clone https://github.com/martomi/chiadog.git
+cd chiadog
+```
+
+3. Create a virtual environment for `chiadog` to run in
+
+```
+# Create a new virtual environment in this folder called 'venv'
+python.exe -m venv venv
+
+# Activate the newly created virtual environment
+. ./venv/Scripts/activate
```

Comment:
Should this not have the opposite slashes / \ ? (I don't know, asking 😄 )
Same about the other occurrences of that.

Reply:
Good catch! That's what I get for typing from memory instead of copy/pasting.